### DISCUSSION TITLE
Do we pay for deactivated users in Kimai?

### DISCUSSION THREAD

**Discussion by EduNetEurope** [kimai/kimai#5131] [Questions & Answers]
Kimai charges on a per-user, per-month basis. Are we required to pay for deactivated users, or does the fee apply only to active users?

**Comment by kevinpapst** [comment]
Cloud / SaaS related questions can be asked at support@kimai.cloud. 
This here is only for development of the core product. 

There are docs available at: https://www.kimai.org/documentation/cloud-payment.html

**Reply by EduNetEurope** [reply]
Thanks, @kevinpapst.

For those who have the same question and find this thread, the answer is in the [linked document](https://www.kimai.org/documentation/cloud-payment.html):

> Active users consume one license.
> Deactivated users do not consume a license. Once deactivated, a license becomes available for another account. If a previously > deactivated user is reactivated, a license is consumed again.

**Reply by kevinpapst** [reply]
What I don't understand in this context: you are not a cloud customer, arent't you.

**Reply by EduNetEurope** [reply]
Not yet but considering if we should switch to cloud from self-hosted.

**Reply by kevinpapst** [reply]
Be my guest, import is for free. Everything else via email please.
